### PR TITLE
Add image placeholders

### PR DESCRIPTION
**Pull request by econdepe**
For empty address book:
image

For no contacts found after search:
image

### CODE REVIEW

File: src/app/components/ContactFieldProperty.js [hunk 1]
```diff
@@ -10,28 +10,27 @@ import { getAllFieldLabels } from '../helpers/fields';
 const ContactFieldProperty = ({ field, value, uid, onChange, ...rest }) => {
     const handleChange = ({ target }) => onChange({ value: target.value, uid });
     const labels = getAllFieldLabels();
-    const val = Array.isArray(value) ? value[0] : value;
```
Comment (EpokK) [inline src/app/components/ContactFieldProperty.js]: Why is this there?

File: src/app/components/ContactsList.js [hunk 2]
```diff
@@ -12,19 +13,41 @@ import { extractMergeable } from '../helpers/merge';
 import ItemCheckbox from './ItemCheckbox';
 import ContactGroupIcon from './ContactGroupIcon';
 import MergeRow from './MergeRow';
-
-const ContactsList = ({ contacts, onCheck, onMerge, user, history, contactID, location }) => {
+import ImportModal from './import/ImportModal';
+import ContactModal from './ContactModal';
+import noContactsImg from 'design-system/assets/img/shared/empty-address-book.svg';
```
Comment (EpokK) [inline src/app/components/ContactsList.js]: External import first.

File: src/app/components/ContactFieldProperty.js (hunk 1 above)
Comment (econdepe) [reply src/app/components/ContactFieldProperty.js]: Are you referring to the line `const val = Array.isArray(value) ? value[0] : value;`?

I guess it was there from the old logic. But that line is not there anymore

Comment (EpokK) [reply src/app/components/ContactFieldProperty.js]: I mean this kind of change should be in an other PR right?

Comment (econdepe) [reply src/app/components/ContactFieldProperty.js]: ouch, yeah, i branched from the wrong branch...

**Pull request merged**